vespaclient-core/src/main/java/com/yahoo/feedhandler/FeedResponse.java

```diff
@@ -162,6 +167,19 @@ private static boolean containsFatalErrors(Stream<Error> errors) {
         return errors.anyMatch(e -> e.getCode() != DocumentProtocol.ERROR_TEST_AND_SET_CONDITION_FAILED);
     }
 
+    private static com.yahoo.container.protect.Error convertErrorCode(int error) {
+        // We should try to enumerate these error a bit finer.
+        // Like busy, no space etc.
+        if (error == DocumentProtocol.ERROR_NO_SPACE) {
+            return com.yahoo.container.protect.Error.INSUFFICIENT_STORAGE;
+        } else if (error >= ErrorCode.TRANSIENT_ERROR && (error < ErrorCode.FATAL_ERROR)) {
+            return com.yahoo.container.protect.Error.INTERNAL_SERVER_ERROR;
+        } if (error >= ErrorCode.FATAL_ERROR && (error < ErrorCode.ERROR_LIMIT)) {
+            return com.yahoo.container.protect.Error.INTERNAL_SERVER_ERROR;
+        }
+        return com.yahoo.container.protect.Error.INTERNAL_SERVER_ERROR;
```

Comment:
3 branches return the same value, so I'm assuming the first 2 are "placeholders" for transient and fatal errors in case we figure out a more appropriate error code for them?

Reply:
Yes, it was to signal that we have different groups.
Transient could return a try again later code, if there are any.